### PR TITLE
Create nc-duckdns.sh

### PR DESCRIPTION
**Pull request by stratacast**
Add script to handle DuckDNS

### CODE REVIEW

File: etc/nextcloudpi-config.d/nc-duckdns.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+configure() 
+{
+
+   if [[ $ACTIVE_ == "yes" ]]; then
+      mkdir $INSTALLPATH 2> /dev/null
+      # Creates duck.sh script that checks for updates to DNS records
+      touch $INSTALLPATH/duck.sh
+      touch $INSTALLPATH/duck.log
+      echo -e "echo url=\"https://www.duckdns.org/update?domains=$DOMAIN_&token=$TOKEN_&ip=\" | curl -k -o $INSTALLPATH/duck.log -K -" > $INSTALLPATH/duck.sh
```
Comment (nachoparker) [inline etc/nextcloudpi-config.d/nc-duckdns.sh]: the field `ip=` is empty, is that correct?

Seems like this downloads a bash script from duckDNS? correct?

File: etc/nextcloudpi-config.d/nc-duckdns.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+DESCRIPTION="Free Dynamic DNS provider (need account from https://duckdns.org)"
+
+install() { :; }
+
+
+configure() 
+{
+
+   if [[ $ACTIVE_ == "yes" ]]; then
+      mkdir $INSTALLPATH 2> /dev/null
```
Comment (nachoparker) [inline etc/nextcloudpi-config.d/nc-duckdns.sh]: I'll change it to `mkdir -p $INSTALLPATH`, which is normally cleaner

**Comment by nachoparker** [comment]
Those notes are so that you understand the changes that I will make in my next commit. They are very small things, the script looks great.

Thank you very much! happy to have a new code contributor! Specially this one request, because duckDNS is the oldest feature request in Github!! #28

File: etc/nextcloudpi-config.d/nc-duckdns.sh (hunk 1 above)
Comment (stratacast) [reply etc/nextcloudpi-config.d/nc-duckdns.sh]: There is no need to enter a value for "ip=" in the URL provided by DuckDNS, they handle all of that for you

The download I believe is to just grab OK or KO from them, which will tell you that DuckDNS is working as it should or not

**Comment by stratacast** [comment]
Thank goodness I've been using it for over a year to know how to contribute this! :)

**Pull request closed without merging**